arm-resources/policy/2016-12-01/swagger/policy.json

```diff
@@ -0,0 +1,701 @@
+          "description": "The type of policy definition. Possible values are NotSpecified, BuiltIn, and Custom.",
+          "enum": [
+            "NotSpecified",
+            "BuiltIn",
+            "Custom"
+          ],
+          "x-ms-enum": {
+            "name": "policyType",
+            "modelAsString": true
+          }
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The display name of the policy definition."
+        },
+        "description": {
+          "type": "string",
+          "description": "The policy definition description."
+        },
+        "policyRule": {
+          "type": "object",
+          "description": "The policy rule."
+        },
+        "parameters": {
+          "type": "object",
+          "description": "The policy parameters. Required if there is a parameter is used in policy rule. "
+        }
+      },
+      "description": "The policy definition properties."
+    },
+    "PolicyDefinition": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PolicyDefinitionProperties",
+          "description": "The policy definition properties."
+        },
+        "id": {
+          "type": "string",
+          "description": "The ID of the policy definition."
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the policy definition. If you do not specify a value for name, the value is inferred from the name value in the request URI."
+        }
+      },
+      "description": "The policy definition.",
+      "x-ms-azure-resource": true
+    },
+    "PolicyDefinitionListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PolicyDefinition"
+          },
+          "description": "An array of policy definitions."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to use for getting the next set of results."
+        }
+      },
+      "description": "List of policy definitions."
+    },
+    "PolicyAssignmentProperties": {
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "The display name of the policy assignment."
+        },
+        "policyDefinitionId": {
+          "type": "string",
+          "description": "The ID of the policy definition."
+        },
+        "scope": {
+          "type": "string",
+          "description": "The scope for the policy assignment."
+        },
+        "parameters": {
+          "type": "object",
+          "description": "The value of the policy parameters. Required if there is a parameter is used in policy rule. "
+        },
+        "description":{
+          "type": "string",
+          "description": "The description of policy assignment. In case of policy violation, end user will see this message. "
```

Comment:
>In case of policy violation, end user will see this message.  [](start = 64, length = 61)

This message will be part of response in case of policy violation.

Also remove blank spaces from the end.